plugins/reporters/static-html/src/main/kotlin/TablesReportModelMapper.kt

```diff
@@ -142,7 +141,7 @@ private fun getProjectTable(input: ReporterInput, project: Project): ProjectTabl
         val detectedLicenses = resolvedLicenseInfo.filter { LicenseSource.DETECTED in it.sources }
             .sortedBy { it.license.toString() }
 
-        val analyzerIssues = projectIssues[id].orEmpty() + analyzerResult.issues[id].orEmpty()
+        val analyzerIssues = projectIssues[id].orEmpty() + input.ortResult.analyzer!!.result.issues[id].orEmpty()
```

Comment:
Thanks to the final `orEmpty()`, `!!` could and should be changed to `?.`. And probably that should even be mentioned as the primary reason for inlining in the commit message.